.github/workflows/install-v2.5-to-ubuntu-20-04.yml

```diff
@@ -1,5 +1,8 @@
 name:  Install v2.5 to Ubuntu 20.04
-on: [push]
+on:
```

Comment:
Let it run in every push because it also checks the server package and it just takes a couple of minutes.